src/rpc/server/scala/serv/src/main/scripts/omega-edit-grpc-server.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
```

Comment:
This file isn't formatted properly. It should follow the same formatting that we use on our TS code

Reply:
I ran `npc prettier -w` on it.  What's not formatted properly?

Reply:
Thats because you don't have a `.prettier.rc` inside of the folder like we do for our TS code. We want all JS/TS files to have same formatting. This file has tabWidth of `4` and semicolons when the rest of the code does not.

.prettier.rc

```rc
{
  "semi": false,
  "singleQuote": true,
  "tabWidth": 2
}
```

Reply:
@scholarsmate Or run it

```bash
npx prettier --tab-width 2 --no-semi --single-quote -w 
```

Reply:
It depends on the application, in this case it doesn't need them and shouldn't have them so that both the TS and JS are the same. This file could very well have a `.ts`  instead of a `.ts` since its a script.